docs/operating-scylla/nodetool-commands/cluster/cleanup.rst

```diff
@@ -0,0 +1,15 @@
+Nodetool cluster cleanup
+========================
+
+**cluster cleanup** - A process that runs in the background and removes data no longer owned by nodes. Used for no tablet tables only.
+
+Running ``cluster cleanup`` on a **single node** cleanup all no tablet tables on all nodes in the cluster.
```

Comment:
"cleanup all" -> "cleans up all"? We probably shouldn't use "cleanup" like this in the docs.

Reply:
I also wouldn't care normally, but this is the user-facing documentation.